web/pandas_web.py

```diff
@@ -487,7 +488,15 @@ def main(
                     )
                 else:
                     body = markdown.markdown(
-                        content, extensions=context["main"]["markdown_extensions"]
+                        content,
```

Comment:
Instead of repeating this here, we should probably add the ecosystem page to this condition above: `if "pdeps/" in fname:`